lib/wordmove/logger.rb

```diff
@@ -2,40 +2,49 @@ module Wordmove
   class Logger < ::Logger
     MAX_LINE = 70
 
+    def initialize(device, strings_to_hide = [])
+      super(device, formatter: proc { |_severity, _datetime, _progname, msg|
+        if strings_to_hide.empty?
+          msg
```

Comment:
Why no new lines before and after `msg` like you did in line 10 ?
Do you want to give some 'padding' to "secretized" strings?

Reply:
Quite sure I forgot those newlines from a debug session; even if they seems stripped (or tests should fail... 🤔 ) I'll take a look and probably remove them.

Ty

Reply:
This is actually green

```ruby
it "will hide the passed strings" do
     expect { logger.info('What I write is hidden') }
          .to output(/What I write is \[secret\]/)
          .to_stdout_from_any_process
end
```

Reply:
I've cleaned the code on line 10